### PR TITLE
Add drivers

### PR DESCRIPTION
**Pull request by ynikolo**
Added drivers for Chrome and Firefox to the project.

### CODE REVIEW

File: configs/Configuration.properties [hunk 1]
```diff
@@ -1,3 +1,2 @@
-driverPath=D:\chromedriver_win32\chromedriver.exe
```
Comment (mdikina) [inline configs/Configuration.properties:1]: Why is this change necessary?

Comment (ynikolo) [reply configs/Configuration.properties:1]: It is not necessary for this change but it was unused and redundant code.

**Review by mdikina — APPROVED** [review]
